scripts/link-check-git-all.sh

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
-(find pages/ content/docs/ src/ .github/ -name '*.md' -o -name '*.js' && ls *.md *.js) \
+repo="$(dirname "$(realpath "$(dirname "$0")")")"
+(find "$repo"/pages/ "$repo"/content/docs/ "$repo"/src/ "$repo"/.github/ -name '*.md' -o -name '*.js' && ls "$repo"/*.md "$repo"/*.js) \
```

Comment:
does it fail if one of the directories does not exist?

Reply:
no - tested (exit status still zero). I presume you're talking about the warning

```
find: ‘...dvc.org/pages/’: No such file or directory
```

Reply:
yes, would be good to catch changes in the docs structure if it's just a matter of adding some flag

Reply:
I assume `pages` would be generated if the site was built. Are you saying it should be removed?

Reply:
no, no ... I'm not even sure if we need to test generated stuff .. just saying that next time someone decides to move `/content` to something like `/static/content` would be nice to detect that change ... again, not a big deal

Reply:
Right. Sounds like something for a different issue. Though it sounds like doing something like a directory move leaved the onus on the mover to update all tests.

Reply:
> like a directory move leaved the onus on the mover to update all tests

correct! but it's better for tests to start failing.